java/org/apache/catalina/webresources/FileResource.java

```diff
@@ -30,10 +29,6 @@
 import java.security.cert.Certificate;
 import java.util.jar.Manifest;
 
-import org.apache.catalina.WebResourceRoot;
-import org.apache.juli.logging.Log;
-import org.apache.juli.logging.LogFactory;
-
```

Comment:
The order of the imports will be checked by checkstyle, so better leave them here.